### PR TITLE
Remove caching and rmq directory, react dep

### PR DESCRIPTION
**Pull request by meagonqz**
Made possible by https://github.com/imperfectproduce/products/pull/416 :confetti_ball: 

I also removed the rmq directory since the only reference to that function is in monkey-wrench (and zen10 fingerlings).

https://github.com/search?q=org%3Aimperfectproduce+getQueue&type=code
https://github.com/search?q=org%3Aimperfectproduce+httpApi&type=code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,16 +27,11 @@
   "dependencies": {
     "babel-preset-env": "^1.6.1",
     "babel-register": "^6.26.0",
-    "bluebird": "^3.5.3",
-    "isomorphic-fetch": "^2.2.1",
     "lodash": "^4.17.21",
     "moment": "^2.22.1",
     "moment-timezone": "^0.5.16",
-    "react": "16.2.0",
```
Comment (meagonqz) [inline package.json:35]: React is removed in [website](https://github.com/imperfectproduce/website/commit/71ec868614e78f632cc3467beb550207d3e26951), [hasselback](https://github.com/imperfectproduce/hasselback/commit/6d5c26f4dd2fb28816653d481969b7abd23728f3#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519), [merch-admin](https://github.com/imperfectproduce/merch-admin/commit/41d1acdeef11d1d27978ab5006e5a4c4895956ae), and [products](https://github.com/imperfectproduce/products/commit/4ff18dc5b639fe16bdffa8e142caf74d8b2bf1f3). Deliveries-admin has it pinned to a [specific published version](https://github.com/imperfectproduce/deliveries-admin/blob/5a0df333e0a50cb673eafca50d66149dc61eb42b/package.json#L34).

We can look to retire monkey-wrench now but it doesn't hurt to clean up these files and deps that we know are dead code.

File: caching/redisKeyValueCache.js [hunk 2]
```diff
@@ -1,117 +0,0 @@
-const redis = require('redis');
```
Comment (meagonqz) [inline caching/redisKeyValueCache.js:1]: added to products in imperfectproduce/products#416

File: rmq/httpApi.js [hunk 3]
```diff
@@ -1,26 +0,0 @@
...
-const getQueue = (queueName, options = {}) => {
-  const {
-    lengthsAge = 60, // 1 minute
-    lengthsIncr = 1 // 1 second
-  } = options;
-
-  const query = `lengths_age=${lengthsAge}&lengths_incr=${lengthsIncr}`;
-  const url = `${baseUrl}/queues/${VHOST}/${queueName}?${query}`;
-
-  return fetch(url)
```
Comment (meagonqz) [inline rmq/httpApi.js:20]: Mainly looked at this bit since it was using fetch (which had a security issue).

**Review by ImperfectNathanielFoster — APPROVED** [review]
LGTM! I agree that react is not needed here.

Verified that
- the rmq directory / files are not referenced elsewhere.
- bluebird, isomorphic-fetch, redis, snappy are no longer referenced after these are removed

**Comment by impf-neldridge** [comment]
👨‍🍳 🤌

**Pull request merged**